include/common/aglGPUMemAddr.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
-#include "agl/detail/aglMemoryPoolHeap.h"
 #include "aglGPUMemBlock.h"
```

Comment:
Should this also get a fully qualified name, or stay like this? What about the ordering in such a case?

Reply:
I personally prefer the styling of not using folders when pulling files from the same folder but if leo wants I can change to include the ``common/`` for consistency